### DISCUSSION TITLE
Is there any plan to support AsyncAPI 3.x ?

### DISCUSSION THREAD

**Discussion by Driky** [stoplightio/spectral#2598] [Q&A]
My team is trying to decide if we will keep a spectral step in our CI pipeline. Since I didn't find any mention of AsyncAPI 3.x in the issues I figured it was the place to ask.

**Comment by msolujic** [comment]
one more interested party here. can we have some ETA for AsyncAPI v3.x support?

**Comment by TribuneX** [comment]
@Driky Is there any alternative around to validate AsyncAPI 3.x files instead of using spectral?
